shared/util/string.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+function toStringForLog(a: any): string {
+  if (typeof a === 'string') {
+    return a
+  } else if (a instanceof Error) {
+    return a.toString()
```

Comment:
`Error.toString()` doesn't include the error stack, which is probably useful information. I suggest building a string out of `a.message` and/or `a.stack` instead